queries.sql

```diff
@@ -132,3 +132,50 @@ SELECT species, AVG(escape_attempts) AS average_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY species;
+
+
+-- Write queries (using join) to answer the following questions:
+-- What animals belong to Melody Pond?
+SELECT a.name
+FROM animals AS a
+JOIN owners AS o ON a.owner_id = o.id
+WHERE o.full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT a.name
+FROM animals AS a
+JOIN species AS s ON a.species_id = s.id
+WHERE s.name = 'Pokemon';
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT o.full_name, COALESCE(array_agg(a.name), '{}'::text[]) AS animals_owned
+FROM owners AS o
+LEFT JOIN animals AS a ON o.id = a.owner_id
+GROUP BY o.full_name;
```

Comment:
- [ ] 👍 Thanks for your effort on the queries. However, since the `owner_id` data in the `data.sql` file hasn't been updated, it's difficult to verify the functionality of this query. Please make sure to update the data to reflect the relationships between animals and owners as required by the project, and if necessary, include a screenshot of the query results to provide a clearer understanding. Keep up the good work! 
- [ ] Please follow this for others query if necessary📊👏

![image](https://github.com/alexansaa/vet_sql/assets/55840999/633147aa-1086-4559-aeda-74b40d367775)